### ISSUE TITLE
[bug] typo in bakta subworkflow

### ISSUE THREAD

**Issue by rpetit3** [bactopia/bactopia#290]
Hi, I'm trying to use bakta but it ended up in the message:

`WARN: Access to undefined parameter 
backta_db -- Initialise it to a default value eg. params.backta_db = some_value`

`Cannot invoke method startsWith() on null object`

`backta_db` also looks like a typo, not sure what went wrong there.

My command was:

`bactopia --wf bakta --bactopia agalactiae/ --bakta_db datasets/bakta/db/` 

Thank you for helping.

_Originally posted by @lfenske-93 in https://github.com/bactopia/bactopia/issues/272#issuecomment-1060904393_

**Comment by rpetit3** [comment]
Hi @ifenske-93

Thank you very much for catching this! I've fixed this typo, and its available on my dev-build. 

```
conda create -n bactopia-dev -c rpetit3 -c conda-forge -c bioconda rpetit3::bactopia
```

This should let you keep moving along, let me know if not!

Cheers,
Robert

**Comment by rpetit3** [comment]
fixed in https://github.com/bactopia/bactopia/releases/tag/v2.0.3

**Comment by rpetit3** [comment]
Apologies @lfenske-93! I thought this was an issue I made!

I've released v2.0.3, which includes a fix for this and should be available on Bioconda today or tomorrow. 

Thank you very much for your help in finding this! I'll keep you posted on replacing Prokka with Bakta!

Cheers, 
Robert
